modules/kn-service-offline-about.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// serverless/cli_reference/kn-offline-services.adoc
+
+[id="kn-service-offline-about_{context}"]
+= About offline mode
+
+Normally, when you execute `kn service` commands, the changes immediately propagate to the cluster. However, as an alternative, you can execute `kn service` commands in offline mode:
+
+. When you create a service in offline mode, no changes happen on the cluster. Instead, the only thing that happens is the creation of the service descriptor file on your local machine.
+. After the descriptor file is created, you can manually modify it and track it in a version control system.
+// Once `update` works, add it here and make it into a list
```

Comment:
does this comment need to be resolved now or is it a TODO for later?

Reply:
It's a TODO for a future release where the `update` command works.